bundle/edu.gemini.qv.plugin/src/main/scala/edu/gemini/qv/plugin/table/renderer/ObservationsRenderer.scala

```diff
@@ -86,9 +87,13 @@ object EncodedObservationsRenderer extends CellRenderer {
     /** Encodes the observation id (i.e. program number and obs id) */
     private def encObsId(o: Obs): String = {
       val pid = o.getProg.getStructuredProgramId
-      Option(pid.getSemester).map(_.takeRight(2)).getOrElse("") +
-        pid.getType +
-        pid.getNumber + "-" +
+      pid.semester.map(_.toString.takeRight(2)).getOrElse("") +
```

Comment:
It seemed odd to me to just take the last two characters of `Semester` but I suppose that's how it is done in the QV?  So "2018A" becomes "8A"?

Reply:
Ugh... that's awful. The last three characters makes sense, but the last two sounds like a terrible idea.